### ISSUE TITLE
NVDA Speech viewer is showing the title of a dialogue but it is not being announced

### ISSUE THREAD

**Issue by NitinPem** [nvaccess/nvda#13562]
### Steps to reproduce:
Open NVDA. Click on a button that opens a page with header.
### Actual behavior:
NVDA speech viewer shows heading level and title, but DOES NOT announce anything
That is, speech viewer shows: 'heading level 2 Create Time Off Request' but it is not being announced
![2022-03-28 21_14_51-Window](https://user-images.githubusercontent.com/102540782/160829956-fc0ccaaf-b6e1-414b-90d1-10545b99e331.png)

### Expected behavior:
NVDA speech viewer shows heading level and title and announces it.
That is, speech viewer shows: 'heading level 2 Create Time Off Request' and also announces it
#### NVDA installed/portable/running from source:
NVDA installed from software center
#### NVDA version:
2021.3.1
#### Windows version:
Windows 10 Enterprise
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
[nvda.log](https://github.com/nvaccess/nvda/files/8379900/nvda.log)

**Comment by NitinPem** [comment]
Hi, I tried to disable browse mode on page load. And issue seems to be fixed

**Comment by feerrenrut** [comment]
Speech is primarily an aid for developers, it doesn't currently replace listening to synthesizer output. There are a variety of ways we could improve the accuracy and usability of the speechviewer, but these are hard to prioritize against features that help our primary users who typically consume synthesizer output or braille. 

Note, that speech viewer shows speech output instantaneously, but a synthesizer takes time to produce audio. Speech can be interrupted by certain events, including moving the mouse with mouse tracking enabled. This can result in the perceived discrepancy you describe.

We can't progress this issue as it stands, we'll need sample code and clear steps to reproduce the problem. At this stage I doubt there is a problem with output from the synthesizer.

For now, I'll close this issue, please comment and provide more info if you'd like it re-opened.
